microsoft-edge/web-platform/sidebar.md

```diff
@@ -0,0 +1,56 @@
+
+* By adapting your existing Progressive Web App (PWA) to run in the sidebar.
+* By making use of the Sidebar API in your Microsoft Edge extension.
+
+
+<!-- ====================================================================== -->
+## Choose the right approach
+
+The two different ways in which you can leverage the sidebar are designed to meet different developer needs. The following are the key differences between sidebar apps and sidebar extensions:
+
+|Approach | Sidebar app | Sidebar extension |
+|---|---|---|
+| Definition | A Progressive Web App (PWA) that runs in the sidebar. | A Microsoft Edge extension that displays content in the sidebar. |
+| Distribution | Like a website, the app is hosted on the app's web server. | The extension is distributed through the [Microsoft Edge Add-ons website](https://microsoftedge.microsoft.com/addons/Microsoft-Edge-Extensions-Home). |
+| Capabilities | The app can make use of all web APIs and features, as well as PWA capabilities such as offline support, push, or file system access. | The extension can make use of powerful extension APIs, such as reading or modifying the current page. |
+| Context | The app runs in its own web context, separate from the main tab. | The extension can access the current webpage and modify the webpage via the extension's content script. |
+| Installation | The user installs the app by interacting with the sidebar when visiting the app's website. | The extension is installed by the user from the [Microsoft Edge Add-ons website](https://microsoftedge.microsoft.com/addons/Microsoft-Edge-Extensions-Home). |
+| User experience | The app has its own icon in the sidebar. Clicking the icon opens the sidebar if it was closed and displays the app's contents. | The user experience depends on the extension. The extension can open the sidebar depending on the websites the user visits in the browser. |
+| Technologies used | Web development and PWA. See [Overview of Progressive Web Apps (PWAs)](../progressive-web-apps-chromium/index.md). | Extension development.  See [Overview of Microsoft Edge extensions](../extensions-chromium/index.md). |
+
+
+<!-- ====================================================================== -->
+## Sidebar apps
+
+Sidebar apps allow you to adapt your existing Progressive Web App (PWA) to securely run alongside other browser tabs for a side-by-side co-browsing experience, which is a great choice for web developers who want to offer companion experiences including social, messaging, or media apps that don't require advanced extensions capabilities.
+
+The sidebar app infrastructure helps users discover and install sidebar apps as they browse the web, providing great discoverability for your app with just a simple change to your web application manifest file. Sidebar apps are a great option if you want to reuse your existing web app for the sidebar in Microsoft Edge and make full use of the web capabilities that are available to PWAs.
```

Comment:
```suggestion
The sidebar app infrastructure helps users discover and install sidebar apps as they browse the web, providing great discoverability for your app just by making a simple change to your web application manifest file. Sidebar apps are a great option if you want to reuse your existing web app for the sidebar in Microsoft Edge and make full use of the web capabilities that are available to PWAs.
```
an alternative to consider

fwiw:
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/u/using-by-using-with - "Don't use with as a synonym for using or by using."

Reply:
incorp'd